helper/chart_data.js

```diff
@@ -2263,3 +2263,255 @@ const createSolutionsArray = async function (data) {
         console.log(err);
     }
 }
+
+
+//Report for entity level report in assessments
+exports.entityLevelReportData = async function (data) {
+
+    return new Promise(async function (resolve, reject) {
+
+        //group the data based on completed date   
+        let groupedSubmissionData = await groupArrayByGivenField(data, "completedDate");
+
+        let completedDateArray = Object.keys(groupedSubmissionData);
+
+        //Sort the date in ascending order
+        completedDateArray = completedDateArray.sort();
+
+        if (completedDateArray.length > config.druid.no_of_assessment_submissions_threshold || 3) {
+            completedDateArray = [sortedDate[0], sortedDate.length - 2, sortedDate.length - 1];
```

Comment:
@deepakr123 -Handle the case for submission count being different, i.e. more than 3

Reply:
resolved